tests/fixture-data/txsource/staking-genesis.json

```diff
@@ -9,5 +9,20 @@
     },
     "fee_split_vote": "1",
     "fee_split_propose": "1"
+  },
+  "total_supply": "90000000",
+  "ledger": {
+    "LQu4ZtFg8OJ0MC4M4QMeUR7Is6Xt4A/CW+PK/7TPiH0=": {"general": {"balance": "10000000"}},
```

Comment:
Make sure that this is enough for really long term tests (it's probably not).

Reply:
i've increased `fundAccountAmount` 100x, and each node account in genesis get's the same amount